cpp/src/ice2slice/Gen.cpp

```diff
@@ -15,21 +15,20 @@ namespace
     static string getCSharpNamespace(const ContainedPtr& cont, bool& hasCSharpNamespaceAttribute)
     {
         // Traverse to the top-level module.
```

Comment:
`m` and `p` pointed to the same Slice element.
The only difference is `m` was more strongly typed.
But this extra typing information wasn't necessary.